### DISCUSSION TITLE
Error while importing in node js

### DISCUSSION THREAD

**Discussion by KrishT0** [ai/nanoid#455] [General]
I was trying to use Nanoid in my application to generate a short ID.
`const { nanoid } = require("nanoid");` used this but while running the app, it shows an error.

Error :
![image](https://github.com/ai/nanoid/assets/78247197/639c31a0-a22f-429c-bac2-21fb5a237b5e)
can anyone tell me what's wrong here?

**Comment by ai** [comment]
You need to migrate your app to ESM or install `nanoid@3` if you are going to use CJS.

**Reply by KrishT0** [reply]
latest version is not available in CJS?

**Reply by ai** [reply]
Not. We are mentioning it in docs.
